doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -286,7 +286,7 @@ Float64
 Half-precision floating-point numbers are also supported (`Float16`), but only as a storage format.
 In calculations they'll be converted to `Float32`:
```

Comment:
This doesn't seem to be true anymore? The following doctest where updated in 79a2e013a28029ab624c8271282e5a13f4d8fdad from Float32 to Float16, but this preceding sentence should have changed too? Perhaps just keep the first sentence:

> Half-precision floating-point numbers are also supported (`Float16`):